CHANGELOG.md

```diff
@@ -27,6 +27,9 @@
 #### Bug Fixes
 - Fixed an issue when using np.where and df.where when the scalar 'other' is the literal 0.
 
+#### Improvements
+- Call getOrCreate at package import time so that sproc user does not need to.
```

Comment:
I would rewrite this as "Make importing Snowpark pandas automatically create a Snowpark session in stored procedures."

Reply:
Good suggestion. Updated.